### ISSUE TITLE
Front-end: Handling of null values in Sped apr?

### ISSUE THREAD

**Issue by mepps** [DC-OSSE/LearnDC_v2#101]
How do we want null values displayed from the sped APR data? Should those indicators not display? Thanks!

**Comment by benjaminrobinson** [comment]
I think this is a similar issue to #95, right?

**Comment by mepps** [comment]
@benjaminrobinson No, this is just for me to know how I should account for null values in this specific exhibit. Nothing you need to do though!

**Comment by cmgiven** [comment]
In the case of 18, 19, and 20, yes, those are state-only and should be hidden. Are you seeing that elsewhere, though? In this case we should confirm during today's meeting. I see some nulls for the targets on 15 for DCPS, for example, but it does have a value.

**Comment by mepps** [comment]
See https://github.com/socialdriver/learndc-data/blob/master/converted/lea/0170/sped_apr.json for an example, where most indicators have null values but a couple have values.

**Comment by cmgiven** [comment]
Gotcha. Ben, do you have any idea why these are nulled? What does the source data show?

**Comment by benjaminrobinson** [comment]
This was survey data collected from LEAs...so sometimes they didn't provide an answer and were left blank so there are a ton of missing values in the dataset (this was why I made the reference to the previous issue, which is similar in that this is how we treat missing values but in that case there was a whole column of data that wasn't doing anything and this is much more spotty in that regard)...

**Comment by cmgiven** [comment]
Gotcha. Then let's definitely raise during the meeting. We probably should note non-reporting in some way.

**Comment by benjaminrobinson** [comment]
I don't think we raised this during the meeting...so what do we want to do about this?

**Comment by gwenrr** [comment]
I now see there are lots of missing values when I look at the staging site. We will have to do some investigation on our end and may put this further off.

**Comment by benjaminrobinson** [comment]
Reasons for missing values are included here:

http://osse.dc.gov/sites/default/files/dc/sites/osse/publication/attachments/FFY%202012%20Public%20Reporting_Revised%20on%2012%209%202014.pdf

Removing missing values drops number of rows in our dataset from 2,124 to 678...Having discussions internally on best way to move forward.

**Comment by gwenrr** [comment]
To anyone who still receives emails from this GitHub: Do you remember this whether we decided not to move forward on publishing the LEA APR data because of this?

**Comment by benjaminrobinson** [comment]
I think this is definitely one of the reasons along with the Equity Report stuff I think we were still dealing with...this just wasn't a priority anymore of the leftover funds we had.
